modules/preview-react/breadcrumbs/lib/Breadcrumbs/Dropdown/Button.tsx

```diff
@@ -22,33 +21,29 @@ export interface DropdownButtonProps extends IconButtonProps {
   toggled: boolean;
 }
 
-export const DropdownButton = createComponent(IconButton)({
+export const DropdownButton = createComponent(TertiaryButton)({
   displayName: 'DropdownButton',
   Component: (
     {
       buttonIcon = relatedActionsIcon,
+      variant = undefined,
       toggled,
```

Comment:
I think we need to fix this. We need to have a `toggled` state for this button.
![](http://g.recordit.co/IBWSwTQ7uX.gif)

Reply:
good find! updating this now!